### PR TITLE
Mobile version started

### PR DESCRIPTION
**Pull request by bekjon-me**
- added header section
- added headline section

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="shortcut icon" href="favicon.ico" type="image/x-icon" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Personal portfolio</title>
+  </head>
+  <body>
+    <header class="container">
+      <div class="logo">
+        <h1>Bekjon</h1>
```
Comment (NduatiKagiri) [inline src/index.html]: - [ ] Kindly use a `a` tag for the page title/logo since it should be a link.

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      </div>
+      <div class="hamburger">
+        <div class="line"></div>
+        <div class="line"></div>
+        <div class="line"></div>
+      </div>
```
Comment (NduatiKagiri) [inline src/index.html]: - [ ] Good job implementing the links inside the header, but kindly wrap the hamburger with a `nav` element. Please note that in your project, the header should contain only the logo and the nav, and then nav links should contained inside the nav.

File: src/index.html [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    </header>
+
+    <section id="headline" class="container">
+      <h2 class="title">Hey There. I’m Bekjon</h2>
+      <p class="subject">I’am a Software Developer</p>
```
Comment (NduatiKagiri) [inline src/index.html]: - [ ] Please note that this should be a heading and not a paragraph. Kindly replace the `p` element with a heading element i.e. `hx`

File: src/index.html [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+            ><img src="./Icon - Twitter.svg" alt="Twitter" />
+          </a>
+        </li>
+        <li>
+          <a href="https://medium.com/"
+            ><img src="./Icon - Medium.svg" alt="Medium" />
+          </a>
+        </li>
+      </ul>
+    </section>
```
Comment (NduatiKagiri) [inline src/index.html]: - [ ] Great! You have done a commendable job on this section

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+  color: #fff;
+}
+
+section#headline > ul.social_media {
+  display: flex;
+  list-style-type: none;
+  column-gap: 16px;
+  padding: 4px;
+  margin-bottom: 120px;
+}
```
Comment (NduatiKagiri) [inline src/style.css:100]: - [ ] Good! You have done a perfect job on your styles

**Review by NduatiKagiri — CHANGES_REQUESTED** [review]
Hi @bekjon-me , 

# CHANGES REQUIRED ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so close

## Highlights ✔️
✅ Perfect User Interface
✅ GitHub Flow observed
✅ Linters Checks are passing
✅ Good documentation

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- For HTML/CSS and HTML/CSS/JS projects, you are allowed, and it's also recommended to create folders for your assets, stylesheets, scripts, and other HTML files, but please always place your `index.html` file in the parent directory. A good reason for this is that `index` files are the entry files to your project. When running your projects, web servers will always check the index files in the parent directory, and if it is not found there, the result will be a `404 error` which means that the requested files/directories can't be found.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Olamarx — APPROVED** [review]
Hi @bekjon-me,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Linters passed ✔️
- Readme file ✔️
- Mobile version ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**